src/Styleguide/Pages/Artist/state.tsx

```diff
@@ -0,0 +1,23 @@
+import { Container } from "unstated"
+
+interface State {
+  isLoggedIn: boolean
+}
+
+export class ArtistAppState extends Container<State> {
+  state = {
+    isLoggedIn: false,
+  }
+
+  logIn() {
+    this.setState({
+      isLoggedIn: true,
+    })
+  }
+
+  logOut() {
+    this.setState({
+      isLoggedIn: false,
+    })
+  }
+}
```

Comment:
Not being used ATM, just a sketch pad

Reply:
How about we try to use local schema extension for this specific state?

Reply:
Oh yeah, I'm curious about this, lets talk after meeting.